net-libs/libnice/libnice-0.1.18.recipe

```diff
@@ -60,25 +63,30 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
+	unset meson
```

Comment:
Not quite sure why this is needed, without it I'm getting an error that the buildtype isn't set (although it is).

Reply:
I added the buildtype first, cleaned ... but still had an error that the buildtype wasn't set (allthough in the summary it showed it was there), without unset meson I couldn't get the build started.

Reply:
compile and install aren't supported by the scriptlet...